specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3431,6 +3431,10 @@
         "dataSourceName": {
           "type": "string",
           "description": "The name of the datasource from which this indexer reads data."
+        },
+		"skillsetName": {
+          "type": "string",
+          "description": "The name of the cognitive skillset executing with this indexer."
```

Comment:
@jeji1101 It would be helpful to mention here that you can leave this null if you want an indexer without an associated skillset.